src/components/common/CertificateBanner.tsx

```diff
@@ -0,0 +1,29 @@
+import { CertificateBannerProps } from '@/interfaces';
+import { FaTrophy, FaLock } from 'react-icons/fa';
+
+const CertificateBanner = ({ totalChapters, completedChapters }: CertificateBannerProps) => {
+  if (completedChapters < totalChapters) {
+    // Locked Banner
+    return (
+      <div className='bg-purple-400 text-white rounded-lg p-4 mt-6 shadow-md flex items-center justify-between'>
+        <div>
+          <h2 className='text-lg font-bold'>Download Certificate</h2>
+          <p className='text-sm'>Complete All to Get Your Certificate.</p>
+        </div>
```

Comment:
Better if we not Hardcode data. We can accept Heading, Subtext and Icon as Props. We should follow it